paasta_tools/cli/cmds/local_run.py

```diff
@@ -74,20 +74,20 @@ def perform_http_healthcheck(url, timeout):
             try:
                 res = requests.get(url)
             except requests.ConnectionError:
-                return (False, "http request failed: connection failed")
+                return False, "http request failed: connection failed"
     except TimeoutError:
-        return (False, "http request timed out after %d seconds" % timeout)
+        return False, "http request timed out after %d seconds" % timeout
 
     if 'content-type' in res.headers and ',' in res.headers['content-type']:
         sys.stdout.write(PaastaColors.yellow(
             "Multiple content-type headers detected in response."
             " The Mesos healthcheck system will treat this as a failure!"))
-        return (False, "http request succeeded, code %d" % res.status_code)
+        return False, "http request succeeded, code %d" % res.status_code
     # check if response code is valid per https://mesosphere.github.io/marathon/docs/health-checks.html
-    elif res.status_code >= 200 and res.status_code < 400:
-        return (True, "http request succeeded, code %d" % res.status_code)
+    elif 200 <= res.status_code < 400:
+        return True, "http request succeeded, code %d" % res.status_code
     elif res.status_code >= 400:
-        return (False, "http request failed, code %d" % res.status_code)
+        return False, "http request failed, code %d" % res.status_code
 
```

Comment:
I just noticed a bug with this function: if the status code is <200, then control drops out the of the if/elif/elif and this function returns None.

https://github.com/Yelp/paasta/issues/678